v22.2/backup-validation.md

```diff
@@ -0,0 +1,143 @@
+1. `RESTORE ... WITH schema_only, verify_backup_table_data`: Run a `schema_only` restore **and** have the restore read all user data from external storage, verify checksums, and discard the data before writing it to disk. To use `verify_backup_table_data`, you must include `schema_only` in the statement. See [Validate the restore of backup table data](#validate-the-restore-of-backup-table-data) for an example.
+
+The options that give the most validation coverage will increase the runtime of the check. That is, `verify_backup_table_data` will take a longer time to validate a backup compared to `check_files` or `schema_only` alone. Despite that, each of these validation options provide a quicker way to validate a backup over running a "regular" restore.
+
+## Validate backup files 
+
+Using `SHOW BACKUP` with the `check_files` option, you can check that all [SST and metadata files](backup-architecture.html) that belong to a backup are present in the storage location.
+
+First, take a backup that we'll use for each of the examples on this page:
+
+{% include_cached copy-clipboard.html %}
+~~~sql
+BACKUP DATABASE movr INTO "s3://bucket?AWS_ACCESS_KEY_ID={Access Key ID}&AWS_SECRET_ACCESS_KEY={Secret Access Key}" AS OF SYSTEM TIME "-10s";
+~~~
+
+To find a specific backup to validate in the storage location, first show the stored backups in the storage location:
+
+{% include_cached copy-clipboard.html %}
+~~~sql
+SHOW BACKUPS IN "s3://bucket?AWS_ACCESS_KEY_ID={Access Key ID}&AWS_SECRET_ACCESS_KEY={Secret Access Key}"; 
+~~~
+~~~
+          path
+------------------------
+  2022/09/19-134123.64
+  2022/09/19-134559.68
+(2 rows)
+~~~
+
+{% include {{ page.version.version }}/backups/check-files-validate.md %}
+
+## Validate the restore of a backup
+
+To validate that a backup is restorable, you can run `RESTORE` with the `schema_only` option, which will complete a restore **without** restoring any user table data. This process is significantly faster than running a [regular restore](restore.html#examples) for the purposes of validation. 
+
+`schema_only` restores produce close to complete validation coverage on backups. However, this restore type does not read or write from any of the SST files, which store the backed-up user table data. You can use `SHOW BACKUP ... WITH check_files` in addition to a `schema_only` restore to check that these files are present for a restore operation. Or, you can use `schema_only` in combination with `verify_backup_table_data`. See [Validate the restore of backup user data](#validate-the-restore-of-backup-table-data).
+
+Run `RESTORE` with either `LATEST` or the specific backup you would like to restore:
+
+{% include_cached copy-clipboard.html %}
+~~~sql
+RESTORE DATABASE movr FROM "2022/09/19-134123.64" IN "s3://bucket?AWS_ACCESS_KEY_ID={Access Key ID}&AWS_SECRET_ACCESS_KEY={Secret Access Key}" WITH schema_only;
```

Comment:
is it worth noting that they would need to delete this DB if they restores it only for testing?

Or that they could restore... with schema_only... with new_db_name = test_db ?

Reply:
Oh yeah, the delete is already included a few paragraphs down. I have dropped the database.

Mentioning the `new_db_name` option is a good point — I will add that with Michael's review